### PR TITLE
New package: simavr-1.6

### PR DESCRIPTION
**Pull request by bobertlo**
gnu-makefile target does not work for this Makefile

### CODE REVIEW

**Comment by bobertlo** [comment]
I forgot a -devel package, but also it needs a musl patch. I will be back.

File: srcpkgs/simavr/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+revision=1
+hostmakedepends="avr-gcc avr-libc pkg-config"
+makedepends="elfutils-devel libfreeglut-devel"
+short_desc="Lean, mean, and hackable AVR simulator"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/buserror/simavr"
+distfiles="https://github.com/buserror/simavr/archive/v${version}.tar.gz"
+checksum=a55ad04d055eef5656c49f78bc089968b059c6efb6a831618b8d7e67a840936d
+CFLAGS+=" -fPIC"
```
Comment (maxice8) [inline srcpkgs/simavr/template]: don't append.

File: srcpkgs/simavr/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (maxice8) [inline srcpkgs/simavr/template]: remove

**Comment by bobertlo** [comment]
@maxice8 ok, changed those. I disabled building the tests because they depend on avr-gcc and don't work in musl.

**Comment by bobertlo** [comment]
I did compile this software and run a couple tests manually on a musl system also. Can't say about arm or anything. The test client is trying to overwrite stderr and musl doesn't like that.

**Pull request closed without merging**